### PR TITLE
PackageManagement API v1 (post-API Review)

### PR DESCRIPTION
**Pull request by DrusTheAxe**
PackageManagement API incorrpoating feedback from API Review. The following functionality is available:
* PackageVolume
* PackageDeploymentManager
  * AddPackage*()
  * EnsurePackage[Set]Ready()
  * IsPackage[Set]Ready()
  * StagePackage*()
  * RegisterPackage*()
* PackageRuntimeManager

NOTE: PackageDeploymentManager requires Windows >= 10.0.19041.0 (aka Win10 20H1)
NOTE: PackageRuntimeManager requires Windows >= 10.0.22621.0 (aka Win11 21H2 aka SV2)

Known Issues:
* Some PackageDeploymentManager methods are not functional yet (throw hresult_not_implemented) awaiting a FrameworkUdk update:
  * IsPackageReadyByUri()
  * EnsurePackageReadyByUriAsync()
  * RemovePackage*Async()
  * ResetPackage*Async()
  * RepairPackage*Async()
  * ProvisionPackage*Async()
  * DeprovisionPackage*Async()
  * IsPackageRegistrationPending()
* PackageVolume.Repair() and .IsRepairNeeded() require an OS patch. These fail if the OS lacks the update.

### CODE REVIEW

File: dev/PackageManager/API/PackageManagerTelemetry.h [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+        {
+            TraceLoggingClassWriteStart(
+                AddPackageByUriAsync,
+                _GENERIC_PARTB_FIELDS_ENABLED,
+                TraceLoggingWideString(packageUri.c_str(), "PackageUri"));
+        }
+        CATCH_LOG()
+        DEFINE_ACTIVITY_STOP(winrt::hstring const& packageUri) noexcept try
+        {
+            TraceLoggingClassWriteStart(
```
Comment (wcheng-msft) [inline dev/PackageManager/API/PackageManagerTelemetry.h:48]: >teStart [](http://example.com/codeflow?start=32&length=7)

should half of these be stops?

**Review by wcheng-msft — APPROVED** [review]
:shipit:

File: dev/PackageManager/API/PackageManagerTelemetry.h (hunk 1 above)
Comment (DrusTheAxe) [reply dev/PackageManager/API/PackageManagerTelemetry.h]: Oops! Good catch! Fixed

**Pull request merged**